scripts/integration.sh

```diff
@@ -20,7 +20,7 @@ reflex run --loglevel debug --env "$env_mode" "$@" & pid=$!
 # Within the context of this bash, $pid_in_bash is what we need to pass to "kill" on exit
 # This is true on all platforms.
 pid_in_bash=$pid
-trap "kill -INT $pid_in_bash ||:" EXIT
+trap "kill -TERM $pid_in_bash ||:" EXIT
```

Comment:
This change was somehow critical to making this test work in WSL.  I.e. it looked as if the action step would hang forever, even beyond the 10 min timeout waiting for ports.

Switching to TERM seems to work ok for WSL and other platforms, as well.